lib/redlock/client.rb

```diff
@@ -210,10 +211,16 @@ def get_remaining_ttl(resource)
 
       private
 
+      def generate_script_shas
```

Comment:
```suggestion
      def generate_script_shas
        # We do not want to load the scripts on every Redlock::Client initialization.
        # Hence, we rely on Redis handing out SHA1 hashes of the cached scripts and
        # pre-calculate them instead of loading the scripts unconditionally. If the scripts
        # have not been cached on Redis, `recover_from_script_flush` has our backs.
```